backend/sirius-web-diagrams-layout/src/main/java/org/eclipse/sirius/web/diagrams/layout/incremental/provider/NodeSizeProvider.java

```diff
@@ -10,19 +10,19 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-package org.eclipse.sirius.web.diagrams.components;
+package org.eclipse.sirius.web.diagrams.layout.incremental.provider;
 
-import java.util.List;
+import java.util.Optional;
 
-import org.eclipse.sirius.web.components.Element;
 import org.eclipse.sirius.web.diagrams.INodeStyle;
 import org.eclipse.sirius.web.diagrams.ImageNodeStyle;
 import org.eclipse.sirius.web.diagrams.ImageNodeStyleSizeProvider;
```

Comment:
The ImageNodeStyleSizeProvider has no place anymore in sirius-web-diagrams and thus the dependency to batik should also be removed from sirius-web-diagrams too

Reply:
That also includes the ImageSizeProvider